### ISSUE TITLE
archives message is automatically unachieved when a new message is sending over - also include ignore msg issue

### ISSUE THREAD

**Issue by adorapham** [FightPandemics/FightPandemics#1908]
I think that should be considered as improvement because later , we will go in to have a very long history message and might have the issue to load the message.
archive-mean something we keep as history  , we can un-achieve to review them later. 
here is what I observation
1-inbox one org Nature has a lot of message for AP - so it decides to achieved this message and agree with i**f you like to read the conversion again, you will have to go to Messages setting to restore it.**
2-after clicking on Achieve button , see the message goes to Achieved  Conversations on Manages Settings.
3-then AP send another message to Nature again after Nature achieved all AP messages
4-Naviage to Nature inbox and see all AP achieved message including the new message in display.

i think we should have another new thread of message from AP to Nature because per achieve conversation display, only when you go to message setting, archived conversation folder to restore if you want to read it.   as time going, the message will get growing and it seems there is no better way to achieve the old messages at the current time.

![achieve](https://user-images.githubusercontent.com/70918430/100043601-9fa18480-2dc2-11eb-8a41-3d10e3c90ed7.gif)

**Comment by rabbi08** [comment]
@adorapham 
This is expected behavior for archived messages as:
From google about archive message behavior:
"When you archive a message: The message will come back to your inbox when someone replies to it." as per Gmail.

**Comment by adorapham** [comment]
> From google about archive message behavior:

@rabbi08 , yes, I see it is for Gmail message but Gmail message has many threads ,not like Direct private message, I just put it in for improvement because as I said the message will be growing and we might face issue of loading them. Developers can review and see if they can do anything about it.

**Comment by adorapham** [comment]
I think it is the same for ignore message. here is what I observation (might be need to get improvement later). At this time, Achieve and Ignore work similar. Achieve has folder while ignore doesn't have folder but the way they come back to inbox is similar.
1-HM send a message request to NM
2-NM take a look and decide to ignore the message request from HM after reading the confirmation dialog box "T**he message request will be deleted from your device. Are you sure you want to ignore?"** 
3-NM clicks on ignore and think the message is deleted
4-HM send another message request to NM
5-NM check the inbox and see all ignored message that NH thought deleted from device still on display plus the new message.
what if NM only want to see the last message but have to keep all the messages that NM decided to ignore and think them should be deleted from the previous action.  NM don't want to block this person , but only want to choose to accept certain msg but seeming have to view all messages among them NM don't feel favor.
![ignore](https://user-images.githubusercontent.com/70918430/100054999-fbc2d380-2dd7-11eb-965c-b9f3a9fc2f21.gif)

**Comment by adorapham** [comment]
as discussion with @Zaydme ,that is something dev will take a look in future.

**Comment by emtay** [comment]
@tejasm2 - this is working as designed right? we should close and can reopen ticket if change mind in future?
